examples/personachat-example.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a07e0f5e",
+   "metadata": {},
+   "source": [
+    "<div>\n",
+    "<img src=\"https://camo.githubusercontent.com/473dd9f992924d27457650251786464f72e54121ac6e9210add0f483ca849277/68747470733a2f2f692e696d6775722e636f6d2f3765523750616e2e706e67\" width=\"40%\">  \n",
+    "</div>\n",
+    "\n",
+    "# Distributed Bloom for Text Generation using Prompt Tuning\n",
+    "\n",
+    "In this example, we showcase how the Bloom model can be efficiently adapted in a decentralized fashion. In particular, servers maintain the Bloom transformer, which is kept unchanged during adaptation, and learn only a few prefix tokens.\n",
```

Comment:
Please write `... in a decentralized fashion using Petals` (where `Petals` is a link leading to https://github.com/bigscience-workshop/petals). This is useful for people referred to this notebook from a random place of the Internet - they'll have a chance to understand what's going on and what the tutorial is talking about.